github/repos.go

```diff
@@ -48,7 +48,8 @@ type Repository struct {
 	Permissions      *map[string]bool `json:"permissions,omitempty"`
 
 	// Only provided when using RepositoriesService.Get while in preview
-	License *License `json:"license,omitempty"`
+	License         *License `json:"license,omitempty"`
+	LicenseTemplate *string  `json:"license_template,omitempty"`
 
 	// Additional mutable fields when creating and editing a repository
```

Comment:
> While creating a new repo, we need to provide a license_template. 

In that case, you should move the new `LicenseTemplate` field into _this_ section, it has all the other fields for creating/editing a repository. Or maybe a sub-section there at the end, since this field is relevant at create-time only, you can't use it when editing an existing repo.

It's not relevant to the `RepositoriesService.Get` section you've added it to currently.

Additionally, you can add `gitignore_template`, which is also missing, if you want. Source: https://developer.github.com/v3/repos/#input.